examples/Docker/docker-compose-kafka.yml

```diff
@@ -0,0 +1,90 @@
+version: '3.8'
```

Comment:
Optional: could I ask you to change examples/README.md to use docker-compose instead k8s-workloads? You would need to also create separate docker-compose file for postgres.

Reply:
Added description how to use docker compose